src/app/projects/project-view/project-view.component.ts

```diff
@@ -10,6 +10,20 @@ export class ProjectViewComponent implements OnInit, AfterViewInit {
   @Input() currentProject = null;
   @Output() projectEvent = new EventEmitter<any>();
 
+  btnStyles = {
```

Comment:
i ovo definisanje button stilova ovde u ovoj komponenti bukvalno nema smisla, a svakako ces to obrisati...